pkg/userdata/amzn2/provider.go

```diff
@@ -140,6 +140,20 @@ package_upgrade: true
 package_reboot_if_required: true
 {{- end }}
 
+{{- if .ProviderSpec.CAPublicKey }}
+- path: "/etc/ssh/sshd_config"
+  content: |
+	TrustedUserCAKeys /etc/ssh/trusted-user-ca-keys.pem
+	CASignatureAlgorithms ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,ssh-ed25519,rsa-sha2-512,rsa-sha2-256,ssh-rsa
```

Comment:
Please move this into a template function and call it in all other places

Reply:
Cool cool